cli/src/lib.rs

```diff
@@ -4305,6 +4464,18 @@ fn get_node_version() -> Result<Version> {
     Version::parse(output).map_err(Into::into)
 }
 
+fn get_recommended_micro_lamport_fee(client: &RpcClient, priority_fee: u64) -> Result<u64> {
+    let fees = client.get_recent_prioritization_fees(&[])?;
+    let fee = fees
+        .into_iter()
+        .fold(0, |acc, x| u64::max(acc, x.prioritization_fee))
+        + 1;
```

Comment:
I don't think this is a good default to have. It basically adds more fees than the required max in the previous 150 slots. The fees will go towards infinity if everyone does this since there is no cap.

Furthermore, paying more fees do not guarantee landing. Median fee would be much better here.

Reply:
I can just let the user define the priority fee, then? I was doing this, since I couldn't get the idl upgrade to work and really wanted to get it through

Reply:
I think it's best to let the user specify it, and if it's not specified, default to the median fee.

Reply:
ok, I addressed this, I thought default would be no priority fees altogether, modified it to default to median fee if no fee specified